dips/dip-172.md

```diff
@@ -14,9 +14,9 @@ This DIP describes the registration step for VASPs to enable eCommerce peer to m
 This DIP builds on previous work in the space of P2M and identity. [DIP-158](https://github.com/diem/dip/blob/main/dips/dip-158.md) describes one approach for supporting P2M leveraging the Diem off-chain protocol defined in DIP-1. DIP-10 defines an on-chain naming service for VASPs, namely the domain portion of a DiemId.
 
 While there might be many P2M flows, three basic elements can be used to construct many of them:
-A flag signifying whether or not the VASP supports the customer flows for P2M payments
-An authentication URL to kick start the peer authentication and authorization within the customer’s VASP of [DIP-158](https://github.com/diem/dip/blob/main/dips/dip-158.md)
-An URL that links to an image representing the VASPs business
+1. A flag signifying whether or not the VASP supports the customer flows for P2M payments
+2. An authentication URL to kick start the peer authentication and authorization within the customer’s VASP of [DIP-158](https://github.com/diem/dip/blob/main/dips/dip-158.md)
+3. A URL that links to an image representing the VASPs business
```

Comment:
quora agrees with you it isn't about the letter that follows but the noise it makes ... stupid stupid stupid english